cmd/minishift/cmd/start.go

```diff
@@ -100,27 +98,27 @@ var (
 		Usage:     "Registry mirrors to pass to the Docker daemon.",
 		Value:     cmdUtil.NewStringSliceValue([]string{}, &[]string{}),
 	}
-)
 
-// startCmd represents the start command
-var startCmd *cobra.Command
+	startCmd *cobra.Command
 
-// Set default value for host data and config dir
-var (
+	// Set default value for host data and config dir
 	hostConfigDirectory  = "/var/lib/minishift/openshift.local.config"
 	hostDataDirectory    = "/var/lib/minishift/hostdata"
 	hostVolumesDirectory = "/var/lib/minishift/openshift.local.volumes"
 	hostPvDirectory      = "/var/lib/minishift/openshift.local.pv"
-)
 
-// clusterUpFlagSet contains the command line switches which needs to be passed on to 'cluster up'
-var clusterUpFlagSet *flag.FlagSet
+	// clusterUpFlagSet contains the command line switches which needs to be passed on to 'cluster up'
+	clusterUpFlagSet *flag.FlagSet
 
-// minishiftToClusterUp is a mapping between flag names used in minishift CLI and flag name as passed to 'cluster up'
-var minishiftToClusterUp = map[string]string{
-	"openshift-env":     "env",
-	"openshift-version": "version",
-}
+	// minishiftToClusterUp is a mapping between flag names used in minishift CLI and flag name as passed to 'cluster up'
+	minishiftToClusterUp = map[string]string{
+		"openshift-env":     "env",
+		"openshift-version": "version",
+	}
```

Comment:
@praveenkumar Any reason it is part of start.go but not in artifact_preflight.go.

Reply:
@LalatenduMohanty it was always part of start.go https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/start.go#L119-L123 I didn't change it and also this is part of start.go because of https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/start.go#L608-L617

Reply:
@praveenkumar cool, I missed that.